xbmc/rendering/dx/DirectXHelper.cpp

```diff
@@ -6,6 +6,7 @@
  *  See LICENSES/README.md for more information.
  */
 
+#include <d3d11.h>
```

Comment:
```suggestion
#include <d3d11_4.h>
```
To be more consistent with others d3d11 includes